app/src/main/java/org/oppia/app/databinding/ImageViewBindingAdapters.java

```diff
@@ -0,0 +1,145 @@
+  public static void setImageDrawable(
+      @NonNull ImageView imageView,
+      @DrawableRes int drawableResourceId
+  ) {
+    imageView.setImageResource(drawableResourceId);
+  }
+
+  /**
+   * Binds the specified [LessonThumbnailGraphic] as the source for the [ImageView]. The view should be specified to have
+   * no width/height (when sized in a constraint layout), and use centerCrop for the image to appear correctly.
+   */
+  @BindingAdapter("android:src")
+  public static void setImageDrawable(
+      ImageView imageView,
+      LessonThumbnailGraphic thumbnailGraphic
+  ) {
+    int drawableResourceId;
+    switch (thumbnailGraphic) {
+      case BAKER:
+        drawableResourceId = R.drawable.lesson_thumbnail_graphic_baker;
+      case CHILD_WITH_BOOK:
+        drawableResourceId = R.drawable.lesson_thumbnail_graphic_child_with_book;
+      case CHILD_WITH_CUPCAKES:
+        drawableResourceId = R.drawable.lesson_thumbnail_graphic_child_with_cupcakes;
+      case CHILD_WITH_FRACTIONS_HOMEWORK:
+        drawableResourceId = R.drawable.lesson_thumbnail_graphic_child_with_fractions_homework;
+      case DUCK_AND_CHICKEN:
+        drawableResourceId = R.drawable.lesson_thumbnail_graphic_duck_and_chicken;
+      case PERSON_WITH_PIE_CHART:
+        drawableResourceId = R.drawable.lesson_thumbnail_graphic_person_with_pie_chart;
+      case IDENTIFYING_THE_PARTS_OF_A_FRACTION:
+        drawableResourceId = R.drawable.topic_fractions_01;
+      case WRITING_FRACTIONS:
+        drawableResourceId = R.drawable.topic_fractions_02;
+      case EQUIVALENT_FRACTIONS:
+        drawableResourceId = R.drawable.topic_fractions_03;
+      case MIXED_NUMBERS_AND_IMPROPER_FRACTIONS:
+        drawableResourceId = R.drawable.topic_fractions_04;
+      case COMPARING_FRACTIONS:
+        drawableResourceId = R.drawable.topic_fractions_05;
+      case ADDING_AND_SUBTRACTING_FRACTIONS:
+        drawableResourceId = R.drawable.topic_fractions_06;
+      case MULTIPLYING_FRACTIONS:
+        drawableResourceId = R.drawable.topic_fractions_07;
+      case DIVIDING_FRACTIONS:
+        drawableResourceId = R.drawable.topic_fractions_08;
+      case DERIVE_A_RATIO:
+        drawableResourceId = R.drawable.topic_ratios_01;
+      case WHAT_IS_A_FRACTION:
+        drawableResourceId = R.drawable.topic_fractions_01;
+      case FRACTION_OF_A_GROUP:
+        drawableResourceId = R.drawable.topic_fractions_02;
+      case ADDING_FRACTIONS:
+        drawableResourceId = R.drawable.topic_fractions_03;
+      case MIXED_NUMBERS:
+        drawableResourceId = R.drawable.topic_fractions_04;
+      default:
+        drawableResourceId = R.drawable.topic_fractions_01;
+    }
+    setImageDrawable(
+        imageView,
+        drawableResourceId
+    );
+  }
+
+  /**
+   * Binding adapter for profile images. Used to either display a local image or custom colored avatar.
+   *
+   * @param imageView View where the profile avatar will be loaded into.
```

Comment:
> Thanks @miaboloix! From a high-level review, please:
> 
> * [ ]  Use 'git mv' to "move" all of the Kotlin files to their Java equivalents (hopefully this is quick--I suggest reverting the Kotlin deletions, copy the Java code, remove the Java file, git mv the Kotlin file to be the Java file, then replace all of the contents). Hopefully this preserves the history and lets us review a delta from Kotlin to Java.
> * [x]  Remove/resolve any TODOs, or turn them into proper issue-tracking TODOs.
> * [x]  Make sure all Java classes have documentation--this can be brief,
> * [x]  Make sure all top-level classes are marked as 'final'.
> * [x]  Make sure that all adapter classes end in 's' to indicate they can contain multiple adapters for naming consistency.
> * [x]  Take a pass on all the Java code to make sure the style is clean & relatively consistent with Kotlin. There are cases of double spaces, incorrectly formatted annotation attributes, etc.
> 
> Otherwise I don't think I have any major concerns with this PR. After seeing the exact changes (e.g. after the 'git mv') I can be more sure.

As we discussed over GVC, git mv wan't able to fully solve the problem (although I think by looking at the individual commits it worked for some files?). I addressed all the other comments, though.

Reply:
Done!